share/backend_config.ini

```diff
@@ -41,3 +41,7 @@ locale = da_DK en_US fi_FI fr_FR nb_NO sv_SE
 [PRIVATE PROFILES]
 #example_profile_2=/example/directory/test2_profile.json
 
+[METRICS]
+# Uncoment the following option to enable the metrics feature
+#statsd_host = localhost
+statsd_port = 8125
```

Comment:
This is the default value so it should probably be
```
# statsd_port = 8125
```

Reply:
Done